server/api/users/models.py

```diff
@@ -49,6 +49,7 @@ class User(AbstractUser):
     username = models.CharField(max_length=20, unique=True)
     first_name = models.CharField(max_length=100, blank=True)
     last_name = models.CharField(max_length=100, blank=True)
+    password = models.CharField(max_length=100)
```

Comment:
I dont think we should store the password in the users model.
Look at elucidates repo for inspiration. https://github.com/codersforcauses/elucidate/tree/main/server/api/apps

You can find the /register in the auth folder and notice how theres no password in their user mode

Reply:
Hi, so do you want us to get rid of the password field in user model or moves the view and url to auth and do something similar to what they did?

Reply:
both, I go to sign up but no user has been created in the django admin dashboard. I have a feeling that having the password as a charfield will store the password in plain text. You might need to speak with Dhruv because he created the user model.

Reply:
Dont worry, I think a user wasnt created because the password I put in my too short and no error displayed in the docker container to let me know. Can you please handle linting errors as well